src/EntryForm.js

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import firebase from "./firebase";
+import {push, ref, getDatabase} from "firebase/database"
+import Swal from "sweetalert2";
+
+const EntryForm = () => {
+
+    const [title, setTitle] = useState("");
+    const [body, setBody] = useState("");
+
+    const date = new Date();
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const year = date.getFullYear();
+    const entryDate = `${day}-${month}-${year}`;
+
+
+    const handleBodyChange = (e) => {
+      setBody(e.target.value);
+    }
+    
+    const handleTitleChange = (e) => {
+      setTitle(e.target.value);
+    }
+  
+    const handleSubmit = (e) => {
+      e.preventDefault();
+
```

Comment:
Some folks might comment on the spaces here but I think it helps make your code more easy to read